packages/components/src/AppLoader/AppLoader.component.js

```diff
@@ -27,7 +27,7 @@ export function AppLoaderComponent({ t }) {
 	);
 }
 AppLoaderComponent.propTypes = {
-	t: PropTypes.func.isRequired,
+	t: PropTypes.func,
```

Comment:
There are a lot of required `t`, but there is a default props value. Removing the `isRequired`